executor/point_get.go

```diff
@@ -246,10 +254,12 @@ func (e *PointGetExecutor) Next(ctx context.Context, req *chunk.Chunk) error {
 
 			// try lock the index key if isolation level is not read consistency
 			// also lock key if read consistency read a value
```

Comment:
Some part of the comments may be changed to adapt the new logic and some may be moved above L240.